packages/injective-cosmwasm/src/derivative_market.rs

```diff
@@ -48,8 +49,7 @@ pub struct DerivativeMarket {
     pub ticker: String,
     pub oracle_base: String,
     pub oracle_quote: String,
-    #[serde(default)]
```

Comment:
Why did you remove this? I believe it was required because 0 is a valid oracle type, but on the Go chain side, the key is omitted from the json if value is zero.

Reply:
put back Default attribute and implemented Default trait for the type with ::Unspecified as default value